blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -305,7 +305,7 @@ public void testHttpAggregatedMultiIngestion_WithMultipleEnumPoints() throws Exc
         verify(context, atLeastOnce()).update(anyLong(), anyInt());
 
         final Locator locator2 = Locator.createLocatorFromPathComponents("99988877", "call_xyz_api");
-        Points<BluefloodEnumRollup> points2 = new FakeEnumAstyanaxReader().getDataToRoll(BluefloodEnumRollup.class,
+        Points<BluefloodEnumRollup> points2 = AstyanaxReader.getInstance().getDataToRoll(BluefloodEnumRollup.class,
                 locator2, new Range(1439231323000L, 1439231325000L), CassandraModel.getColumnFamily(BluefloodEnumRollup.class, Granularity.FULL));
         Assert.assertEquals(2, points2.getPoints().size());
 
```

Comment:
We definitely need more integration tests to test out the new query for enum values features implemented in this story.